Controller/Kount/Ens.php

```diff
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+
+namespace Magento\Braintree\Controller\Kount;
+
+use Magento\Braintree\Model\Kount\EnsConfig;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\HTTP\PhpEnvironment\RemoteAddress;
+
+/**
+ * Class Index
+ */
+class Ens extends Action
+{
+    const KOUNT_MERCHANT_ID = 'payment/braintree/kount_id';
+    /**
+     * @var EnsConfig
+     */
+    private $ensConfig;
+    /**
+     * @var RemoteAddress
+     */
+    private $remoteAddress;
+
+    /**
+     * Index constructor.
+     *
+     * @param Context $context
+     * @param EnsConfig $ensConfig
+     * @param RemoteAddress $remoteAddress
+     */
+    public function __construct(
+        Context $context,
+        EnsConfig $ensConfig,
+        RemoteAddress $remoteAddress
+    ) {
+        parent::__construct($context);
+        $this->ensConfig = $ensConfig;
+        $this->remoteAddress = $remoteAddress;
+    }
+
+    /**
+     * @return ResponseInterface|ResultInterface
+     * @throws LocalizedException
+     * @throws \Exception
+     */
+    public function execute()
+    {
+        $response = $this->resultFactory->create(ResultFactory::TYPE_JSON);
+
+        if (!$this->isAllowed()) {
+            $response->setHttpResponseCode(401);
+        }
+
+        $request = file_get_contents('php://input'); // @codingStandardsIgnoreLine
```

Comment:
Does `$this->getRequest()->getContent()` work?

Reply:
It does! Thank you :) I used `php://input` as it was used somewhere else in the module. I will update shortly.